### PR TITLE
bump erl elixir

### PR DESCRIPTION
**Pull request by InoMurko**
Bump the elixir and erlang versions, go full rust for crypto stuff.
Bumps Rocksdb, make sure you install it.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -19,12 +19,12 @@ executors:
...
   builder:
     docker:
-      - image: omisegoimages/elixir-omg-builder:stable-20200410
+      - image: omisegoimages/elixir-omg-builder:dev-da1ecc8
     working_directory: ~/src
 
   builder_pg:
     docker:
-      - image: omisegoimages/elixir-omg-builder:stable-20200410
+      - image: omisegoimages/elixir-omg-builder:dev-da1ecc8
```
Comment (ayrat555) [inline .circleci/config.yml:27]: are you planning to merge this pr when the corresponding pr in omisego-image is merged?

File: .circleci/config.yml [hunk 2]
```diff
@@ -34,7 +34,7 @@ executors:
 
   builder_pg_geth:
```
Comment (ayrat555) [inline .circleci/config.yml:35]: I saw you bumped geth to 1.9.15 in https://github.com/omisego-images/docker-elixir-omg/pull/45/files#diff-093788b95b8bce4248ae369334936680088f911fc96463fc973308a091cceaacR48.
will you bump in this repo?

File: .circleci/config.yml [hunk 3]
```diff
@@ -44,10 +44,25 @@ executors:
...
 
 commands:
+
+  add_rust_to_path:
+    description: "Add path to PATH env var"
+    steps:
+      - run:
+          name: Add rust to PATH env
+          command: echo 'export PATH=~/.cargo/bin/:$PATH' >> $BASH_ENV
+  install_rust:
```
Comment (ayrat555) [inline .circleci/config.yml:58]: can we specify rust v 1.44 as you did in https://github.com/omisego-images/docker-elixir-omg/pull/45/files#diff-5097196aa827816e990d13cafdea11a1ff82823cee1699a6d74a006d84765eefR66 ?

**Review by ayrat555 — APPROVED** [review]

File: .circleci/config.yml (hunk 3 above)
Comment (InoMurko) [reply .circleci/config.yml:58]: I don't know how to do that with rustup, do you know?

Comment (ayrat555) [reply .circleci/config.yml:58]: `rustup default ...` should work

Comment (InoMurko) [reply .circleci/config.yml:58]: not sure this is gonna work:
```
inomurko@DESKTOP-63PDNVC:/opt/omg-childchain-v1$ rustup default 1.46.0
info: syncing channel updates for '1.46.0-x86_64-unknown-linux-gnu'
info: latest update on 2020-08-27, rust version 1.46.0 (04488afe3 2020-08-24)
error: some components unavailable for download for channel 1.46.0: 'miri', 'rust-analyzer-preview'
If you require these components, please install and use the latest successful build version,
which you can find at <https://rust-lang.github.io/rustup-components-history>.

After determining the correct date, install it with a command such as:

    rustup toolchain install nightly-2018-12-27

Then you can use the toolchain with commands such as:

    cargo +nightly-2018-12-27 build
inomurko@DESKTOP-63PDNVC:/opt/omg-childchain-v1$ echo $?
1
```

File: apps/omg/mix.exs [hunk 4]
```diff
@@ -31,10 +31,11 @@ defmodule OMG.MixProject do
 
   defp deps() do
     [
-      {:ex_plasma, git: "https://github.com/omisego/ex_plasma.git", ref: "83db9a0f5a441f7e59ae23f8c64658e2c387f8a5"},
+      {:ex_plasma, git: "https://github.com/omisego/ex_plasma.git", ref: "5e94c4fc82dbf26cb457b30911505ec45ec534ea"},
```
Comment (achiurizo) [inline apps/omg/mix.exs:34]: we should be able to use hex.pm now?

**Review by achiurizo — APPROVED** [review]

**Review by DanielaIvanova — APPROVED** [review]

File: apps/omg/mix.exs (hunk 4 above)
Comment (InoMurko) [reply apps/omg/mix.exs:34]: I don't think we can.

File: bin/rocksdb [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  echo $sys
+  case $sys in
+    Linux*)
+      install_linux_rocksdb
+      ;;
+    Darwin*)
+      install_rocksdb_source
+      ;;
+    *)
+      fancy_echo "Unknown system"
```
Comment (achiurizo) [inline bin/rocksdb:43]: [fancy_echo](https://github.com/omgnetwork/omg-childchain-v1/blob/4c26cc2abce1f0ec14b8381efe3a882abb07b1ce/bin/setup#L3) isn't availabile in `bin/rocksdb`

**Pull request merged**